File: Modul 4/Python/PRAK403-2410817310013-AhmadAlifPratamaPutra.py
data = input().split()
angka1 = int(data[0])
angka2 = int(data[1])

if angka1 < angka2:
    for i, j in zip(range(angka1, angka2 + 1), range(angka2, angka1 - 1, -1)):
        print(f"{i} {j}", end=" - ")
else:
    for i, j in zip(range(angka1, angka2 - 1, -1), range(angka2, angka1 + 1)):
        print(f"{i} {j}", end=" - ")

File: Modul 4/c/PRAK403-2410817310013-AhmadAlifPratamaPutra.c
#include <stdio.h>
int main() {
    int angka1, angka2, i, j;

    scanf("%d %d", &angka1, &angka2);

    if (angka1 < angka2) {
        for (i = angka1, j = angka2; i <= angka2; i++, j--) {
            printf("%d %d - ", i, j);
        }
    } else {
        for (i = angka1, j = angka2; i >= angka2; i--, j++) {
            printf("%d %d - ", i, j);
        }
    }
    return 0;
}